uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/sub-controllers/toil-request-ctrl.js

```diff
@@ -50,6 +51,10 @@ define([
           return $q.reject(vm.error);
         }
 
+        if (vm.request.toil_expiry_date) {
```

Comment:
Please add the comment here, which you added in the description, so that we know which branch is doing what.